packages/template-retail-react-app/app/utils/test-utils.js

```diff
@@ -135,11 +155,13 @@ export const TestProviders = ({
                         fetchedToken={bypassAuth ? (isGuest ? guestToken : registerUserToken) : ''}
                     >
                         <CurrencyProvider currency={DEFAULT_CURRENCY}>
```

Comment:
@bfeister and I talked about combining IntlProvider and CurrencyProvider - the latter is a part of the former and I don't see a need to have them seperated.